spec/std/isa/inst/Zvks/vsm3me.vv.yaml

```
@@ -10,7 +10,10 @@ long_name: No synopsis available
 description: |
   No description available.
 definedBy:
-  anyOf: [Zvks, Zvksh]
+  extension:
+    anyOf:
+      - name: Zvks
+      - name: Zvksh
```

Zvks requires Zvksh.

```suggestion
  extension:
    name: Zvksh
```